.repo-metadata.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "bigtable-hbase",
+  "name_pretty": "Google Cloud Bigtable Hbase Client",
```

Comment:
This also doesn't seem in line with the native client

Reply:
This is only used for README generation, which we are skipping